### ISSUE TITLE
`"nullable": true` not parsed

### ISSUE THREAD

**Issue by RemyDekor** [StefanTerdell/json-schema-to-zod#15]
First, thank you for this library!

Currently, `"nullable": true` is not parsed, even though there is `z.nullable()` in zod.

I know we can represent `null` in Json-Schema using a primitive *value* (similar to javascript) such as:
`"type": ["string", "null"]`.

### Example:
input:
```
{
    "properties": {
        "name": {
            "type": ["string", "null"]
        },
        "surname": {
            "type": "string",
            "nullable": true,
        },
    },
    "required": ["name", "surname"],
    "title": "Person",
    "type": "object"
}
```

output:
```
const schema = z.object({
  name: z.union([z.string(), z.null()]),
  surname: z.string(),
});
```
---
But my use-case is that an OpenAPI backend is generating the Json-schemas for the payloads I get in the front-end.

And, from my understanding, OpenAPI happens to consider `null` as a value, and not a type.
So it generates Json-Schemas with this `"nullable"` field, which features in their ["extended subset of JSON Schema Specification"](https://swagger.io/docs/specification/data-models/keywords/).

I'm still learning Typescript, but I think `json-schema-to-zod` currently uses `JSONSchema7` coming from `@types/json-schema`, which does not mention this `"nullable"` field.

I'm not sure how I should go about that.

**Comment by StefanTerdell** [comment]
As you guessed the open api spec is not supported right now. If you feel up for coding a PR, you could add a check for the nullable prop at the top of the parseSchema switch which would use a parser that returns something like `${parseSchema({...schema, nullable: undefined})}.nullable()`.

**Comment by StefanTerdell** [comment]
"Fix" released in 0.3.0 :)
